crates/storage/src/lazy/mapping.rs

```diff
@@ -164,6 +180,45 @@ const _: () = {
 mod tests {
     use super::*;
 
+    /// A dummy type which `REQUIRES_DEEP_CLEAN_UP`
+    #[derive(PartialEq, Debug, scale::Encode, scale::Decode)]
+    struct DeepClean<T>(T);
```

Comment:
I wasn't able to find any of the "default" types that we implement `SpreadLayout` for which had `REQUIRES_DEEP_CLEAN_UP` set to `true`. Let me know if there is such a type so I can use that instead

Reply:
`ink_storage::Box` is an example.

Generally `ink_storage::Box` is _the_ interface to work with `SpreadLayout` types from within `PackedLayout` cells.

Reply:
I tried using a `Box`, but [it uses the `REQUIRES_DEEP_CLEAN_UP` of `T`](https://paritytech.github.io/ink/src/ink_storage/traits/impls/prims.rs.html#237), which are all `false` for primitives types as far as I could tell

Reply:
You usually use ink_storage::Box with SpreadLayout types that do nit implement PackedLayout. For example other ink_storage::collections types such as ink_storage::Vec.